src/model/ValidatorFactory.js

```diff
@@ -26,8 +26,19 @@ export default class ValidatorFactory {
 
   static _isQuestionAnswered(answerValue) {
     if (Array.isArray(answerValue)) {
-      return answerValue.length > 0;
+      return (
```

Comment:
This method contains a lot of functionalities related to JSON-LD utils. It's best to move some part of this implementation there. Moreover, even FormUtils is a feasible place to put such functionality there ...

Reply:
Yes you are right, I did not notice I updated it, it is much better now